CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # New Relic Ruby Agent Release Notes
 
+## v9.0.0
+
+- **Enabled Thread Instrumentation by default**
+
+  The configuration option `instrumentation.thread.tracing` is now enabled by default. Allowing the agent to properly monitor code that occurs inside of threads. If you are currently using custom instrumentation to start a new transaction inside of threads, this may be a breaking change, as it will no longer start a new transaction if one already exists.
```

Comment:
The sentence "Allowing the agent to properly monitor code that occurs inside of threads" seems like it's missing something? Did you mean to put a comma after default? Or could you something to the beginning, like "This instrumentation allows..."?

Also, could you start the description under the version number as well?

Reply:
Ahh yes, thank you! I missed that. I've changed both of these things in a33b7e91d565389e4ff6fc75b98d391f4624693b